src/views/catalog/CreateFromInstanceTypes/components/SelectInstanceTypeSection/SelectInstanceTypeSection.tsx

```diff
@@ -26,8 +32,8 @@ const SelectInstanceTypeSection: React.FC = () => {
             onSelect={handleSelect}
             customData={{
               category: instanceType,
-              selectedCategory: selectedCategory,
-              selectedSize: selectedSize,
+              selectedCategory: selectedInstanceType.category,
```

Comment:
Optional chaining operators here and below?

Reply:
added, thanks :)